src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
+    List<FruitTransaction> getFruitList(List<FruitTransaction> list);
```

Comment:
The method signature suggests that it returns a new list of 'FruitTransaction' objects, but it is not clear what the difference is between the input list and the returned list. The interface should have a clear contract indicating what this method does with the input list.